src/renderers/shared/stack/reconciler/__tests__/ReactErrorBoundaries-test.js

```diff
@@ -84,12 +103,15 @@ describe('ReactErrorBoundaries', function() {
       }
     }
 
-    var EventPluginHub = require('EventPluginHub');
     var container = document.createElement('div');
-    EventPluginHub.putListener = jest.fn();
     container.innerHTML = ReactDOMServer.renderToString(<Boundary />);
     expect(container.firstChild.innerHTML).toBe('Happy Birthday!');
-    expect(EventPluginHub.putListener).not.toBeCalled();
+    // note: renderToString does not call componentDidMount
```

Comment:
In most cases it's fine to just test client rendering since the server codepath is mostly a subset of the client one.